__tests__/app.test.js

```diff
@@ -150,6 +150,16 @@ describe("GET /api/articles", () => {
         })
       })
   })
+  test("200: responds with articles filtered by topic", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        articles.forEach((article) => {
+          expect(article.topic).toBe("mitch")
+        })
+      })
+  })
```

Comment:
Nice first test, but we should also have a test for when we are given a valid topic that exists but one that has no associated articles:
E.g `/api/articles?topic=paper`

What should be the expected response for this?